themes/socialblue/src/Plugin/Preprocess/Page.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Drupal\socialblue\Plugin\Preprocess;
+
+use Drupal\socialbase\Plugin\Preprocess\Page as PageBase;
+
+/**
+ * Pre-processes variables for the "page" theme hook.
+ *
+ * @ingroup plugins_preprocess
+ *
+ * @BootstrapPreprocess("page")
+ */
+class Page extends PageBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function preprocess(array &$variables, $hook, array $info) {
+    parent::preprocess($variables, $hook, $info);
+
+    $style = theme_get_setting('style');
+    if ($style && $style === 'sky') {
```

Comment:
The first check is unneccessary here I think. Any value that would return false there would also not `=== 'sky'`.